### DISCUSSION TITLE
Is it possible to install `GD` with `jpeg` support?

### DISCUSSION THREAD

**Discussion by Jeckerson** [shivammathur/setup-php#445] [Q&A / Help] [answered]
It seems, not sure completely, that by installing `gd` extension, it won't enable `jpeg` support.

Also, it is possible to specify flags for extension installation, for example:
```
extensions: gd --with-jpeg-dir=/usr
```

**Comment by shivammathur** [comment, marked as answer]
@Jeckerson 
Yes, in next release you will be able to compile `gd` from source with the required configure options using setup-php.
Currently this feature is in develop branch.

Test workflow: https://github.com/shivammathur/test-setup-php/actions/runs/768848306/workflow

More details here.
https://github.com/shivammathur/setup-php/discussions/418

**Reply by shivammathur** [reply]
@Jeckerson This is now released in `2.11.0`.
